assets/src/stories-editor/components/index.js

```diff
@@ -15,6 +15,7 @@ export { default as FontFamilyPicker } from './font-family-picker';
 export { default as RotatableBox } from './rotatable-box';
 export { default as PreviewPicker } from './preview-picker';
 export { default as Inserter } from './inserter';
+export { default as MediaInsider } from './media-inserter';
```

Comment:
Typo: should be `MediaInserter`

Reply:
Good spot. Resolved in https://github.com/ampproject/amp-wp/pull/3093/commits/7f1f2b9155f9869acb0ae027c8a31512ca8d589e